src/test/java/org/mockito/internal/util/TimerTest.java

```diff
@@ -38,14 +37,9 @@ public void should_return_false_when_time_run_out() throws Exception {
         Assertions.assertThat(timer.isCounting()).isFalse();
     }
 
-    @Test
+    @Test(expected = FriendlyReminderException.class)
     public void should_throw_friendly_reminder_exception_when_duration_is_negative() {
-        try {
-            new Timer(-1);
-            Assert.fail("It is forbidden to create timer with negative value of timer's duration.");
```

Comment:
We should not remove the check for the expected error message, ExpectedException-Rule to the rescue.

Reply:
@ChristianSchwarz nothing is removed here - `fail`'s argument is the error message that's printed once the statement is reached, it doesn't validate anything.

Regardless - using `ExpectedException` to verify the error message would make this a better test.
I'll revise the patch to include it.